### DISCUSSION TITLE
11.0.0 beta - Earthstar × Willow

### DISCUSSION THREAD

**Discussion by sgwilym** [earthstar-project/earthstar#335] [Announcements]
Good morning! The day for an Willow-powered Earthstar 11 beta is finally here. This is a big experiment: a new protocol underpinning everything, complementary new APIs, and of course, the greatest adventure of them all: getting it all to work in JavaScript.

Another experiment is how Earthstar will be distributed. A beta release of `@earthstar/earthstar` is available from JSR. https://jsr.io/@earthstar/earthstar@11.0.0-beta.2. You can find installation instructions for your favourite runtime (yes, Node too) on there. Hopefully JSR will save us from lots of cross-runtime publishing headaches, let’s see.

The README is very much in progress (https://github.com/earthstar-project/earthstar/tree/willow), but there is already very descriptive API documentation available in your IDE and viewable here: https://jsr.io/@earthstar/earthstar@11.0.0-beta.2/doc. A great place to start is the `Peer` class: https://jsr.io/@earthstar/earthstar@11.0.0-beta.2/doc/~/Peer

This beta phase serves the purpose of making sure @earthstar/earthstar can really be used to build apps for the browser, Deno, and Node; that the APIs are as ergonomic as they should be; and to get the docs up to snuff. Your feedback — no matter how nitpicky — is now key! Issues, ideas, and PRs are all welcome on the Earthstar repo.

<hr /><em>This discussion was created from the release <a href='https://github.com/earthstar-project/earthstar/releases/tag/11.0.0-beta.2'>11.0.0 beta - Earthstar × Willow</a>.</em>

**Comment by midnight-wonderer** [comment]
Hi,
I see the software has been stuck in beta for a while now, and the focus seems to have shifted to the Willow protocol.
I'm interested in developing on Earthstar; I wonder if v11 will be API-compatible with v10?

Do you have any recommendations regarding a new application? Should I start with v11 beta?

**Reply by sgwilym** [reply]
Hi @midnight-wonderer,

v11 will not be compatible with v10, as it will use Willow underneath. Starting with the v11 beta will give you a better idea of the new data model and how it will affect your application.
